### ISSUE TITLE
saved tabs (remembered session) are incorrect

### ISSUE THREAD

**Issue by monkeydri** [elementary/files#440]
When I open pantheon-files it reloads the tabs I had opened but not the ones from my last session, as far as I remember those are the one from the first time I launched pantheon-files (first time I closed it).

Interestingly the dconf entry (org > pantheon-files > preferences > tab-info-list) is different than the tabs reopened on launch, but is also incorrect (not the last tabs I used before closing pantheon-files).

- pantheon-files from git source, commit 74b8c8880ff9e5380d1c1896b2e64266c7fd103e (had same bug with version from ubuntu repository)
- ubuntu 18.04 beta (had same bug with 17.10)
- gnome-shell v3.28.1

**Comment by jeremypw** [comment]
This is interesting.  I have tried to reproduce this on Juno by switching between different user sessions and opening and closing Files but the tabs are restored correctly (on a per user basis) in each case.

As you are using a recent commit, it will look for settings in io/elementary/files/preferences/tab-info-list.  So you need to have compiled *and* installed the application to install the correct schema. 

Please confirm whether you experience this issue after taking this into account.
You may have a better experience if you install from the elementary repositories.

**Comment by monkeydri** [comment]
Ohh, my bad I was looking at the wrong dconf entry indeed ! org > pantheon > files > preferences > tab-info-list is holding correct values

I tried to reproduce the issue (version from  commit 74b8c88) :

- check inital content of dconf org > pantheon > files > preferences > /tab-info-list : `[(2, '/mnt/data', 'file:///mnt/data/web'), (2, '/mnt/data/dri/code/', 'file:///mnt/data/dri/code/android/flashtool-firmwares')]`
- open a set of different tabs (pantheon-files was already opened), close window.
- check content of dconf org > pantheon > files > preferences > /tab-info-list : correctly updated : `[(2, '/home/dri', 'file:///home/dri'), (2, '/mnt/data', 'file:///mnt/data'), (2, '/home/dri/Downloads', 'file:///home/dri/Downloads')]`
- reopen pantheon-files : tabs from last session are correctly reopened.

Bottom line : it is all in my head, no bug there. Because I had this issue with a previous version of pantheon-files or maybe because I hard rebooted and tabs could not be saved in dconf entry I assumed that the bug was still there.

*ps : Thankyou Jeremy from taking the time to answer my issue (by the way elementaryOS is awesome but I like to customize things a lot so I stick with gnome/ubuntu for now but I will give it anoher shot soon).*
